sdks/java/core/src/test/java/org/apache/beam/sdk/io/FileBasedSinkTest.java

```diff
@@ -417,6 +419,89 @@ public void testGenerateOutputFilenamesWithoutExtension() {
     assertEquals(expected, actual);
   }
 
+  @Test
+  public void testBuildTemporaryDirectoryName() {
+    // Tests for local files without the scheme.
+    assertThat(
+        new FileBasedWriteOperation.TemporaryDirectoryBuilder()
```

Comment:
Extract `new FileBasedWriteOperation.TemporaryDirectoryBuilder()` into a local variable and reformat?